{
  try {
    Assert.assertEquals('a',(int)this.extractor.getIntValue(this.bean));
  }
 catch (  final Exception e) {
    fail("Should not throw exception");
  }
}
